<?xml version="1.0" encoding="UTF-8"?>
<!-- 
//
// Copyright (c) 2021 The University of Aizu
// Open source under the BSD License.
//
 -->
<ui version="4.0">
 <class>Database</class>
 <widget class="QDialog" name="Database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="All">
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="BB">
        <property name="text">
         <string>Black_body</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="OilBB">
        <property name="text">
         <string>Oil_bath_black_body</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="Winselwan">
        <property name="text">
         <string>Winselwan(meteorite)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="MARAr">
        <property name="text">
         <string>MARA_rock_plate</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="Murray">
        <property name="text">
         <string>Murray(meteorite)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="Murchison">
        <property name="text">
         <string>Murchison(meteorite)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="Sky">
        <property name="text">
         <string>Sky</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ColliBB">
        <property name="text">
         <string>Colli_BB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Instruments </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Target tempareture</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="targetTl"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>to Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="targetTh"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Bolometer Temperature </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bolTl"/>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>to Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bolTh"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Package Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pkgTl"/>
        </item>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>to Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pkgTh"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Case Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="caseTl"/>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>to Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="caseTh"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Shutter Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="shutTl"/>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>to Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="shutTh"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Lens Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lensTl"/>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>to Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lensTh"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="dbSearchButton">
        <property name="text">
         <string>Search from Database</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>171</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Search Result List</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="dbSearchList"/>
    </item>
   </layout>
  </widget>
  <widget class="ShowImage" name="thumbnailView">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>494</width>
     <height>276</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>320</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="loadLogButton">
      <property name="text">
       <string>Load searched log file</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="crearLogButton">
      <property name="text">
       <string>Clear all searched log file</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="showInfoButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Show info</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="subPair">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>320</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightValue">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="widthValue">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pixelValue">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="showGraph">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Y line graph</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setTIRDirectory">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>390</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Set TIR data directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="connectState">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>400</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: </string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShowImage</class>
   <extends>QOpenGLWidget</extends>
   <header>showimage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
